docs/user-guide/kubectl/kubectl.md

```diff
@@ -86,6 +86,7 @@ kubectl
 * [kubectl create](kubectl_create.md)	 - Create a resource by filename or stdin
 * [kubectl delete](kubectl_delete.md)	 - Delete resources by filenames, stdin, resources and names, or by resources and label selector.
 * [kubectl describe](kubectl_describe.md)	 - Show details of a specific resource or group of resources
+* [kubectl doc](kubectl_doc.md)	 - Print model's documentation through swagger's API.
```

Comment:
I think we can do without specifying "swagger" here. Many kubectl users might not know what swagger is and dont need to.
If we decide to change this to kubectl get kind, then we can say "Show the fields of a given kind"